mtp_prisoner_location_admin/settings/base.py

```diff
@@ -23,6 +31,8 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
+TEMPLATE_DEBUG = True
```

Comment:
Yep, there are a few things that need to be setup properly for production I think.

Probably best as a separate task.